bin/fastatic

```diff
@@ -19,8 +20,12 @@ fastatic({
 		dest: program.dest,
 		logLevel: program.logLevel
 	})
-	.then(config => logger.log('Done. Optimised static files in', config.dest))
+	.then(output => {
+		const { fileSize, dest } = output;
+		filesizeReporter(fileSize, program.logLevel);
```

Comment:
Be consistent, with either `filesize` or `fileSize` (I think `filesize` is more common). You are mixing them in variable and function names.

The reporter is about filesizes, so it makes sense the first parameter is `filesize`. I can however imagine the reporter being extended at some point and then we may want to pass more options. So instead of passing `logLevel` directly as second argument I would change the signature to:
`filesizeReporter(filesizes, { logLevel: program.logLevel });`